catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStorePublisher.java

```diff
@@ -210,7 +210,7 @@ public void handleEvent(Event event) {
                 (RegistryStore) bundleContext.getService(((ServiceEvent) event.getProperty(
                         EventConstants.EVENT)).getServiceReference());
 
-        if (!previousAutoPush && registryStore.isAutoPush()) {
+        if (registryStore.isAutoPush()) {
```

Comment:
This will now publish to the remote registry on any update if AutoPush is enabled and the remote registry has not already been published to (this is checked in the publish function).